dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -369,6 +369,32 @@ class DeleteAccountsHelperTest < ActionView::TestCase
     assert_nil section_b.code
   end
 
+  #
+  # Table: dashboard.facilitators_workshops
+  #
+
+  test 'deletes facilitators_workshops rows for purged facilitator' do
+    workshop = create :workshop
+    facilitator = workshop.facilitators.first
+
+    refute_empty facilitator.workshops_as_facilitator
+    result = ActiveRecord::Base.connection.exec_query(<<~SQL)
+      SELECT * FROM facilitators_workshops WHERE facilitator_id = '#{facilitator.id}';
+    SQL
```

Comment:
I'm checking this table manually because it doesn't have a corresponding ActiveRecord model.